docs/theme/css/extra.css

```diff
@@ -0,0 +1,19 @@
+:root > * {
+  --md-code-hl-color: rgb(255, 255, 255, 0.05);
+}
+
+.md-banner .discord {
+    margin-inline-start: 0.2em;
+}
+
+.md-typeset :is(.twemoji) {
+    display: inline-flex;
+    height: px2em(18px);
+    vertical-align: text-top;
+}
+
+.md-typeset svg {
+      width: px2em(18px);
```

Comment:
Those `px2em` things are invalid (not sure why are those needed but if they are then probably some pre-processing is missing).